### DISCUSSION TITLE
好耶，也是成功当上绳匠了

### DISCUSSION THREAD

**Discussion by Pray-Fay** [share121/inter-knot#2900] [其他]
我的性别是纸扎投石机（欸嘿~）
![6F17D26F5411269CD742763D7E589FA8](https://github.com/user-attachments/assets/6c43e8c4-8e30-4806-adfe-7a28c159c81e)

**Comment by seelesee** [comment]
祝八不郎夫妇99<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
